doc/config.rst

```diff
@@ -0,0 +1,53 @@
+Configuration File Formats
+============================
+
+Petitboot is capable of parsing multiple flavors of configuration files for gathering the required information
+for booting the next kernel. GRUB2 and SYSLINUX configuration formats are both supported along with a "native"
+format. Documentation for both the `GRUB2 <https://www.gnu.org/software/grub/manual/grub/grub.html#Configuration>`_ 
+and `SYSLINUX <https://wiki.syslinux.org/wiki/index.php?title=Config>`_
+configuration formats can be found in their respective projects documentation.
```

Comment:
We also support kboot and yaboot formats, if that's worth mentioning.

The syslinux parsing is most commonly used for pxe-style booting, which might be worth a mention, eg:

    and SYSLINUX <...> (typically used for PXE configuration)

Reply:
For those configuration formats I wasn't able to find great webpages to link to for describing them more in detail, perhaps it could be mentioned just in passing without more detail? I'm inclined to leave them out personally.

I did add the note regarding the SYSLINUX use.